ui/src/views/user/settings/SettingsView.vue

```diff
@@ -4,78 +4,181 @@
     tag="section"
   >
     <v-row justify="center">
-      <v-col
-        cols="10"
-        md="10"
-      >
-        <material-card
-          color="primary"
+      <v-col :cols="isMobile? 12 : 8">
```

Comment:
Thank you for the good work!
However, some changes might be needed. Please check below code and apply what you may see fit. 

```
<template>
  <v-container
    id="user-settings-view"
    tag="section"
  >
    <v-row justify="center">
      <v-col cols="12">
        <v-card
          class="user-settings-view__card"
          :class="{'pa-10': !isMobile}"
          :elevation="isMobile? 0 : 2"
        >
          <v-row justify="center">
            <v-col :cols="isMobile ? 12: 8">
              <center>
                <v-img
                  class="mx-2 mb-10"
                  :src="require('@assets/svg/profile_picture.svg')"
                  width="200"
                />
              </center>

              <ValidationObserver v-slot="{ handleSubmit }">
                <v-form
                  v-model="isFormValid"
                  class="user-settings-form"
                  @keyup.native.enter="handleSubmit(onFormSubmit)"
                  @submit.prevent="handleSubmit(onFormSubmit)"
                >
                  <v-container class="py-0">
                    <v-row>
                      <v-col
                        cols="12"
                        class="pa-0"
                      >
                        <ValidationProvider
                          v-slot="{ errors }"
                          name="E-mail"
                          rules="required|email"
                        >
                          <v-text-field
                            v-model.trim="userEmail"
                            outlined
                            required
                            disabled
                            :error-messages="errors[0]"
                            label="E-mail"
                            class="pa-0 user-settings-form__email"
                          />
                        </ValidationProvider>
                      </v-col>
                      <v-col
                        cols="12"
                        class="pa-0"
                      >
                        <ValidationProvider
                          v-slot="{ errors }"
                          name="Password"
                          rules="required|min:8"
                        >
                          <v-text-field
                            v-model.trim="formData.password"
                            outlined
                            required
                            hint="Minimum 8 characters"
                            persistent-hint
                            label="Password"
                            class="pa-0 "
                            :append-icon="isPasswordShown ? 'mdi-eye' : 'mdi-eye-off'"
                            :type="isPasswordShown ? 'text' : 'password'"
                            :error-messages="errors[0]"
                            @click:append="isPasswordShown = !isPasswordShown"
                          />
                        </ValidationProvider>
                      </v-col>
                      <v-col
                        cols="12"
                        class="mt-10"
                      >
                        <v-row justify="center">
                          <v-btn
                            block
                            height="53"
                            min-width="150"
                            class="mt-2 mb-5 v-btn--primary"
                            :disabled="!isPasswordLengthSuitable"
                            @click="handleSubmit(onFormSubmit)"
                          >
                            SAVE SETTINGS
                          </v-btn>
                        </v-row>
                        <v-row justify="center">
                          <v-btn
                            text
                            min-width="150"
                            class="text-none pa-0 mt-2 mb-5 user-settings-form__delete-button"
                            @click="onDeleteAccount()"
                          >
                            Delete account
                          </v-btn>
                        </v-row>
                      </v-col>
                    </v-row>
                  </v-container>
                </v-form>
              </ValidationObserver>
            </v-col>
          </v-row>
        </v-card>

        <modal
          v-if="!isMobile"
          v-model="isDeleteItemModalShown"
          name="delete-modal"
          title="We are sorry to see you go!"
          content="By deleting your account, you will lose all of your surveys and survey responses. If you are sure what you are doing, click the delete button below to continue."
          primary-action-button-text="YES, I WANT TO DELETE MY ACCOUNT"
          @click:primary-action="onDeleteConfirmation"
        />

        <BottomSheet
          v-else
          v-model="isDeleteItemModalShown"
          name="delete-bottomsheet"
          title="We are sorry to see you go!"
          content="By deleting your account, you will lose all of your surveys and survey responses. If you are sure what you are doing, click the delete button below to continue."
          primary-action-button-text="YES, I WANT TO DELETE MY ACCOUNT"
          @click:primary-action="onDeleteConfirmation"
        />

        <MaterialSnackbar
          v-model="isSuccessSnackbarShown"
          type="success"
          timeout="500"
        >
          New settings have been saved!
        </MaterialSnackbar>
      </v-col>
    </v-row>
  </v-container>
</template>

<script>
import { mapGetters } from "vuex";

export default {
  name: "UserSettingsView",
  data() {
    return {
      isFormValid: false,
      isPasswordShown: false,
      formData: {
        email: "",
        password: "",
      },
      isDeleteItemModalShown: false,
      isSuccessSnackbarShown: false,
    };
  },
  computed: {
    ...mapGetters("user", ["userData"]),
    userEmail() {
      return this.userData.email;
    },
    isPasswordLengthSuitable() {
      return this.formData.password.length >= 8;
    },
  },
  methods: {
    onFormSubmit() {
      // to do: post form data to back-end's update user api
      this.isSuccessSnackbarShown = true;
    },
    onDeleteAccount() {
      this.isDeleteItemModalShown = true;
    },
    onDeleteConfirmation() {
      // to do: call delete api
      this.isDeleteItemModalShown = false;
      this.$store.dispatch("user/setUserData", {});
      this.$store.dispatch("user/setToken", "");
      this.$store.dispatch("user/setItems", []);
      this.$cookies.remove("user");
      this.$nextTick(() => {
        setTimeout(() => {
          this.$router.push({ name: "general-user-delete-thankyou" });
        }, 500);
      });
    },
  },
};
</script>

<style lang="scss">
.user-settings-form {
  &__email {
    &.v-text-field--outlined > .v-input__control > .v-input__slot {
      background-color: $light-gray;
    }
  }

  &__delete-button {
    letter-spacing: 0;
    font-weight: 400;
    text-transform: none;
    text-decoration: underline;
  }
}
</style>

```